### PR TITLE
Add IntelliJ metadata to the Python projects to fix initial loading issue

### PR DESCRIPTION
**Pull request by etanshaul**
## What this PR does

Adds the intellij metadata (in the `.idea` folder) to version control.

## Why?

Given the current state of the python samples, when opened in IntelliJ for the first time, the following appears:
image

The project isn't correctly detected, and the user remains in this odd state.

Closing and re-opening the project fixes things, then the user sees the correct structure (and can use Cloud Code):
image

Obviously this is not ideal.

By committing the metadata, and having it available to the IDE on first load, this problem is fixed, and the project loads properly on the first try and can immediately use Cloud Code features.

Note: Tested working in Idea Ultimate (with the python plugin), and in Pycharm

### CODE REVIEW

**Comment by etanshaul** [comment]
note: a similar fix will be needed for `Go` and possibly (not-yet-confirmed) for `Node.js`. Will go one by one per language.

**Comment by etanshaul** [comment]
@patflynn as fyi since I think you may have been running into something similar

File: .templates/template_location.json [hunk 1]
```diff
@@ -3,14 +3,26 @@
     "repoPath": "https://github.com/GoogleCloudPlatform/cloud-code-samples.git",
     "directoryPath": "python/python-hello-world",
     "templatePath": "python-hello-world",
-    "templateName": "Python: Hello World"
+    "templateName": "Python (Flask): Hello World"
```
Comment (simonz130) [inline .templates/template_location.json]: Could you please rebase your local branch so template_location.json is not part of your change?
I wanted to submit this change separately (addition of django)

File: python/django/python-hello-world/.idea/misc.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.6" project-jdk-type="Python SDK" />
```
Comment (simonz130) [inline python/django/python-hello-world/.idea/misc.xml]: what is this?
Is it native intellij setting? Should we maintain this when/if we change the version of python supported in the templates?

File: .templates/template_location.json (hunk 1 above)
Comment (etanshaul) [reply .templates/template_location.json]: Oops. Didnt catch that. Will update.

File: python/django/python-hello-world/.idea/misc.xml (hunk 2 above)
Comment (etanshaul) [reply python/django/python-hello-world/.idea/misc.xml]: Good question. Let me test without it to see if it still works

**Comment by etanshaul** [comment]
@simonz130 ptal

Merged from upstream, and removed the `misc.xml` files.

**Comment by etanshaul** [comment]
@googlebot I fixed it..

**Comment by etanshaul** [comment]
@simonz130 could you please merge for me once approved? I don't have permissions

File: python/django/python-hello-world/.idea/misc.xml (hunk 2 above)
Comment (etanshaul) [reply python/django/python-hello-world/.idea/misc.xml]: fixed. files removed

File: .templates/template_location.json (hunk 1 above)
Comment (etanshaul) [reply .templates/template_location.json]: fixed. merged from upstream

**Pull request merged**